jetty-server/src/test/java/org/eclipse/jetty/server/ProxyProtocolTest.java

```diff
@@ -57,6 +62,45 @@ public void destroy() throws Exception
             server.stop();
     }
 
+    @Test
+    public void testNoProxyJustHttp() throws Exception
```

Comment:
If the goal of this test is to test whether the absence of the PROXY protocol lands on the next protocol, then why wrapping TLS into a detector?

Otherwise wrapping TLS is tested and documented already (at least in 10).
See `DetectorConnectionTest`.

If it is an example, it is "wrong" as the `SslConnectionFactory` is not configured with a KeyStore so in case of TLS it will never work.

Reply:
The point with that specific testcase is to show that proxy and ssl are optional connection factories.
I'll be happy to document that on that specific testcase.

Do you want another testcase showing that ssl works too?

Reply:
@joakime I don't understand.
How's this test different from `DetectorConnectionTest.testOptionalSsl()`?